### DISCUSSION TITLE
Is it possible importing components?

### DISCUSSION THREAD

**Discussion by mybardaklar** [posthtml/posthtml-components#38] [Q&A]
Hello @thewebartisan7,

Is it possible importing components? Because i dont wanna use it like "<module src="../components/button.html".

Do you know Astro.js? Probably you've heard of it. I want it something like that.
Link: https://docs.astro.build/en/core-concepts/astro-components

As you can see it allows us importing components and using them. And also it allows us declaring local variables and importing files.

Is it possible something like this?

Thank you.

**Comment by thewebartisan7** [comment]
Hi, I am not sure to understand your question.
Can you provide some example?

With posthtml-components you don't include your module like you show, but you can use `<x-tag>` as explained in the docs.

If you mean to import like an ES6 module then that it's not possible.
